Content.Server/GameObjects/Components/Movement/SlipperyComponent.cs

```diff
@@ -90,6 +91,11 @@ public void CollideWith(IEntity collidedWith)
             if (percentage < IntersectPercentage)
                 return;
 
+            if(!EffectBlockerSystem.CanSlip(collidedWith))
+            {
+                return;
+            }
```

Comment:
For consistency with the code above

```suggestion
                return;
```